### ISSUE TITLE
why trans.Normalize([0.5, 0.5, 0.5], [0.5, 0.5, 0.5])

### ISSUE THREAD

**Issue by tengshaofeng** [TreB1eN/InsightFace_Pytorch#21]
hi, you really did a great job. Here I have a question. 
why trans.Normalize([0.5, 0.5, 0.5], [0.5, 0.5, 0.5]) not     mean = [0.485, 0.456, 0.406]
    stdv = [0.229, 0.224, 0.225]  ?
can you help me?

**Comment by TreB1eN** [comment]
mean = [0.485, 0.456, 0.406]
stdv = [0.229, 0.224, 0.225] 
where is this statistics coming from ? Imagenet pretraining ?
Well, I think this work is training with faces from scratch, it doesn't related to other pretrained paras

**Comment by tengshaofeng** [comment]
so mean = [0.485, 0.456, 0.406]
stdv = [0.229, 0.224, 0.225] is also ok?

**Comment by TreB1eN** [comment]
it also gonna work I guess. But the Best way is to calculate the maen and std of your Training Data
